src/statmodels.jl

```diff
@@ -162,6 +162,18 @@ predict(obj::RegressionModel) = error("predict is not defined for $(typeof(obj))
 predict!(obj::RegressionModel) = error("predict! is not defined for $(typeof(obj)).")
 dof_residual(obj::RegressionModel) = error("dof_residual is not defined for $(typeof(obj)).")
 
+"""
+    params(model)
+
+Return all parameters of a model.  For a linear model, this would return coefficients as
+well as any other parameters.
+"""
+params(obj) = error("params is not defined for $(typeof(obj))")
+
+"""
+Write the parameters of a model into preallocated storage.
```

Comment:
My reasoning for moving params from Distributions to StatsBase is so that LearnBase isn't importing Distributions for a single name.  I'm actually indifferent to whether StatsBase exports it or not.